docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -81,6 +83,11 @@ Property Name                                      Description
                                                    By default, the value corresponds to the default protocol version
                                                    used in the underlying Cassandra java driver.
                                                    Possible values include ``V3``, ``V4``, ``V5``, ``V6``.
+
+``cassandra.load-policy.use-dc-aware``             Set to ``true`` to use ``DCAwareRoundRobinPolicy``, defaults
```

Comment:
So thats weird... its required but it has a default value? So can I leave it out and then it will just use the default? If that is the case.. its NOT required..

Reply:
It has default value, but we need to enable it explicitly. We will consider changing the default value later.

Reply:
so this is NOT required then... lets remove it form the code snippet earlier

Reply:
As we talked offline, changing the default value (false → true) is also fine to me.

Reply:
At the time of this writing, we use

https://github.com/trinodb/trino/blob/9f5357cfc788c67dab72ebda7b63b5cf3c16ea7d/plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraClientModule.java#L130

as the sole load balancing policy class:

> should almost always be used; it requires a local datacenter to be specified either programmatically when creating the session, or via the configuration option: datastax-java-driver.basic.load-balancing-policy.local-datacenter. It can also use a highly efficient slow replica avoidance mechanism, which is by default enabled – see the option: datastax-java-driver.basic.load-balancing-policy.slow-replica-avoidance.